src/os/bluestore/BlueFS.cc

```diff
@@ -3685,6 +4008,9 @@ void BlueFS::append_try_flush(FileWriter *h, const char* buf, size_t len)/*_WF_L
   bool flushed_sum = false;
   {
     std::unique_lock hl(h->lock);
+    if (h->file->is_new_wal() && h->get_buffer_length() == 0) {
+      h->wal_header_filler = h->append_hole(sizeof(uint64_t));
```

Comment:
sizeof(uint64_t) is meaningless here. To be replaced with the actual type or head_size() call

Reply:
Done.